src/Payments/Payment.php

```diff
@@ -155,14 +157,14 @@ final class Payment extends DateAwareEntity {
      * @JMS\SerializedName("currency_date")
      * @JMS\Groups({"response"})
      */
-    private $currency_date;
+    private $currencyDate;
```

Comment:
Po pobraniu z API u mnie to pole jest NULL-em:
```
  ["currencyDate":"Webit\WFirmaSDK\Payments\Payment":private]=>
  NULL
```
więc zadeklarowany powinien być typ: `@var ?\DateTime`.

Analogicznie z PaymentAmount::for... przyjdą NULL-e do pozostałych pól $currency...

Reply:
+1